assets/collage.css

```diff
@@ -397,7 +397,7 @@
   left: 0;
   overflow: auto;
   width: 100%;
-  background: rgba(var(--color-base-text-rgb), 0.2);
+  background: var(--color-foreground-20);
```

Comment:
Not a fan of these "percentage" variables. I think `rgba(var(--color-base-text-rgb), 0.2)` is more readable. What do we gain by using variables that mimic `rgba`?

Reply:
Mostly to make the CSS a bit less verbose
``` css
div {
  color: rgba(var(--color-foreground-rgb), 0.2);
}

div {
  color: var(--color-foreground-20);
}
```

But yes I think we can use the former instead of the latter if it's less ambiguous

Reply:
@Thibaut Updated. [See details](https://github.com/Shopify/dawn/pull/97#issuecomment-875043775)